prog-o-meter.py

```diff
@@ -143,23 +150,30 @@ def add_day(self):
         self.days_remaining = self.GOAL - self.days
         self.completion_date = self.get_completion_date(self.days_remaining)
         self.canvas.itemconfig(self.rectangle_list[self.days-1], fill = "green")
+        self.canvas.itemconfig(self.encourage_text, text = "".join(self.encourage[self.new_no(self.x)]))
         update_days_file(self.filename, self.days)
         self.canvas.itemconfig(self.countdown_text, text = "".join(("You have ", str(self.days_remaining), " days left!\n\n", "If you code everyday, you will be done with this project on ", self.completion_date)))
-        if self.days >=self.GOAL:        # Disable add_day_button if goal have been reached 
-            self.add_day_button.config(state = "disabled")             
-
+        if self.days >=self.GOAL:        # Disable add_day_button if goal have been reached
+            self.add_day_button.config(state = "disabled")
+            self.canvas.itemconfig(self.hog, text=("".join(("Congrats! ", self.username))))
+    def new_no(self, x):
+    	new = randint(0, 9)

```

Comment:
I think it would be good with an inline comment explaining what this line does. 
After the closing parenthesis, add 8 spaces and then # YOUR_COMMENT_HERE
The comment can just be something simple, like "get a random integer between 0-9"

Reply:
Do i write it about the line 160?